README.md

```diff
@@ -22,6 +22,7 @@ JSDoc linting rules for ESLint.
         * [`require-description-complete-sentence`](#eslint-plugin-jsdoc-rules-require-description-complete-sentence)
         * [`require-example`](#eslint-plugin-jsdoc-rules-require-example)
         * [`require-hyphen-before-param-description`](#eslint-plugin-jsdoc-rules-require-hyphen-before-param-description)
+        * [`require-identifier`](#eslint-plugin-jsdoc-rules-require-identifier)
```

Comment:
You've updated generated README file. See https://github.com/gajus/eslint-plugin-jsdoc/tree/master/.README.

Reply:
Oh, sorry. Will be fixed after fully review from your side 👍

Reply:
All the rest is good. :-)

Reply:
Fixed and divided into two commits 🎉